tiledb/tests/test_util.py

```diff
@@ -107,3 +108,58 @@ def test_array_exists(self):
         self.assertFalse(tiledb.array_exists(uri3, ctx=ctx))
         with tiledb.from_numpy(uri3, np.arange(0, 5), ctx=ctx) as T:
             self.assertTrue(tiledb.array_exists(uri3, ctx=ctx))
+
+    def test_ls(self):
+        def create_array(array_name, sparse):
+            dom = tiledb.Domain(
+                tiledb.Dim(name="rows", domain=(1, 4), tile=4, dtype=np.int32),
+                tiledb.Dim(name="cols", domain=(1, 4), tile=4, dtype=np.int32),
+            )
+            schema = tiledb.ArraySchema(
+                domain=dom, sparse=sparse, attrs=[tiledb.Attr(name="a", dtype=np.int32)]
+            )
+            if sparse:
+                tiledb.SparseArray.create(array_name, schema)
+            else:
+                tiledb.DenseArray.create(array_name, schema)
```

Comment:
```suggestion
            tiledb.Array.create(array_name, schema)
```

It will pick up whether to create a sparse or dense array from the passed in schema.